bin/localstack/mk-s3

```diff
@@ -1,36 +1,12 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 # DOC: Create S3 bucket in localstack.
 
-pushd $(git rev-parse --show-toplevel) > /dev/null
+source bin/lib.sh
+source bin/localstack/lib.sh
 
-set -e
+create_s3_bucket="s3api create-bucket \
+--bucket civiform-local-s3 \
```

Comment:
I believe you don't need the trailing `\`s here

Reply:
done